Tests/Engine/DataFeeds/LiveTradingDataFeedTests.cs

```diff
@@ -424,13 +424,13 @@ public void Unsubscribes()
                 }
                 else
                 {
-                    // should of remove trade and quote bar subscription and split/dividend/delisting for trade bar
-                    Assert.AreEqual(currentSubscriptionCount - 5, _dataQueueHandler.SubscriptionDataConfigs.Count);
+                    // should of remove trade and quote bar subscription and split/dividend/delisting for both (8)
+                    Assert.AreEqual(currentSubscriptionCount - 8, _dataQueueHandler.SubscriptionDataConfigs.Count);
                     // internal subscription should still be there
                     Assert.AreEqual(0, _dataQueueHandler.SubscriptionDataConfigs
                         .Where(config => !config.IsInternalFeed)
                         .Count(config => config.Symbol == Symbols.SPY));
-                    Assert.AreEqual(1, _dataQueueHandler.SubscriptionDataConfigs.Count(config => config.Symbol == Symbols.SPY));
+                    Assert.AreEqual(4, _dataQueueHandler.SubscriptionDataConfigs.Count(config => config.Symbol == Symbols.SPY));
```

Comment:
1 -> 4
there are internal subscriptions to aux data (splits/dividend/delisting)